tests/unit/utils/envVars.go

```diff
@@ -0,0 +1,14 @@
+package utils
```

Comment:
Can you rename the file to either `envvars.go` or `env_vars.go` to be consistent with naming conventions.